.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+- Related Issue #
+- Closes #
+
+## Testing Instructions
+
+_Please replace this line with instructions on how to test your changes, a note
+on the OS and python versions this has been tested on, as well as any relevant
+images/screenshots._
+
+## Added/updated tests?
+_We encourage you to write unit tests when possible._
```

Comment:
```suggestion
<!-- We encourage you to write unit tests when possible. -->
```
Maybe we change this to a comment so contributors still see it when filling out the template but it doesn't appear when published. It also seems like the convention established here is that italic text should be replaced. This line isn't necessarily something to be replaced.